packages/components/pie-textarea/src/index.ts

```diff
@@ -95,7 +104,9 @@ export class PieTextarea extends FormControlMixin(RtlMixin(LitElement)) implemen
         this._internals.setFormValue(this.value);
     }
 
-    protected firstUpdated (): void {
+    protected firstUpdated (_changedProperties: PropertyValues): void {
+        super.firstUpdated(_changedProperties);
```

Comment:
as I know, it's [not necessary](https://lit.dev/docs/components/lifecycle/#firstupdated) to call `super` inside `firstUpdated`